### ISSUE TITLE
Listing of Classes Within Curricula

### ISSUE THREAD

**Issue by KCoterwas** [Capping-CPCA/Web-App#27]
It would be great if the classes within each curriculum to be listed in either the week order it is offered or in alphabetical order.

**Comment by algozine** [comment]
For clarity, they should be listed in the order that they specify .. not alpha .. a curriculum has a set order for the PEP program

**Comment by blackjack26** [comment]
That would require them to add the classes in the proper order. We have no order associated with these classes so they wouldn’t be able to reorder the classes if they made a mistake. If we want that it’s another column on the curriculum classes table like `rank` or something.

**Comment by algozine** [comment]
This order of classes within a curricula doesn't seem to be functioning quite right yet.

Use case: I was adding the 4 courses for Cornerstone on .37, and all was going well until I added the 4th class - "Communication" and it put it at the top of the list instead of the bottom.  The other 3 are in the correct order that I entered them.
![image](https://user-images.githubusercontent.com/31901115/33961710-078c7f9c-e01d-11e7-83a7-7de8c0412e8c.png)
